sr_robot_launch/src/sr_robot_launch/mock_sr_ur_robot_hw.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2021 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+# Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import rospy
+import actionlib
+import actionlib_tutorials.msg
+import control_msgs
+from control_msgs.msg import FollowJointTrajectoryAction, FollowJointTrajectoryResult
+from std_msgs.msg import Bool
+from ur_dashboard_msgs.srv import (GetSafetyMode, GetSafetyModeResponse, GetProgramState,
+                                   GetRobotMode, Load, IsProgramRunning, GetProgramStateResponse,
+                                   GetRobotModeResponse, LoadResponse, IsProgramRunningResponse)
+from ur_dashboard_msgs.msg import SafetyMode, ProgramState, RobotMode
+from std_srvs.srv import Trigger, TriggerResponse
+
+
+class STATE(object):
```

Comment:
convention for class name is `ClassName`

Reply:
8f98e0a1f6e877ce860e15b368831277ee173d5a